x/concentrated-liquidity/math.go

```diff
@@ -52,3 +54,20 @@ func calcAmount1Delta(liq, sqrtPriceA, sqrtPriceB sdk.Dec) sdk.Dec {
 	diff := sqrtPriceB.Sub(sqrtPriceA)
 	return liq.Mul(diff)
 }
+
+func computeSwapStep(sqrtPriceCurrent, sqrtPriceTarget, liquidity, amountRemaining sdk.Dec, lte bool) (sqrtPriceNext sdk.Dec, amountIn sdk.Dec, amountOut sdk.Dec) {
```

Comment:
does `lte` represent if we're moving up or down in the x axis? Can we document this to make it clearer?